test/RPCSession.cpp

```diff
@@ -30,8 +30,30 @@
 using namespace std;
 using namespace dev;
 
+
 IPCSocket::IPCSocket(string const& _path): m_path(_path)
 {
+#if defined(_WIN32)
+	m_socket = CreateNamedPipe(
```

Comment:
This seems to be the server-side part, but we need to implement the client-side part.
We should use `CreateFile` (but not actually create the file if it does not exist, please take a look at the docs) and then use `TransactNamedPipe` for sending and receiving data and `CloseHandle` for closing it.